app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -235,7 +236,7 @@ class MainActivity :
     private val bottomSheetQueue: MutableList<(() -> Unit)?> = mutableListOf()
 
     override val coroutineContext: CoroutineContext
-        get() = Dispatchers.Default
+        get() = Dispatchers.Main
```

Comment:
Don't change this `coroutineContext`. It will have impact on other coroutines launched in the `MainActivity` which might be doing some heavy operations.